concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1085,14 +1085,18 @@ jobs:
                   . ./config/config.sh
                   echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
 
-                  . ./config/quota_settings.sh
-                  ${QUOTA_non_basic_services_allowed} || prefix="dis"
-
-                  cf update-quota "${QUOTA_DEFAULT}" \
-                    -m "${QUOTA_memory_limit}"M \
-                    -r "${QUOTA_total_routes}" \
-                    -s "${QUOTA_total_services}" \
-                    --${prefix}allow-paid-service-plans
+                  for quota_file in config/*_quota.sh; do
+                    (
```

Comment:
👍  for the subshell.

The syntax is quite subtle and easy to overlook though. What do you think about refactoring the contents of the subshell out to a script and calling it with `env -i` to have the same effect?

I'm not sure what the behaviour would be if one of the environment variables was empty because someone omitted or typoed a key in the quota hash. Would it fail or set an unlimited quota? It'd be good to `set -u` to catch that (although L1090 would need refactoring or excluding).